CODEOWNERS

```diff
@@ -0,0 +1,48 @@
+* @m-mirz @dinkelbachjan
+/.clang-format @stv0g @m-mirz
+/.devcontainer @stv0g @m-mirz
+/.dockerignore @stv0g
+/.editorconfig @stv0g @m-mirz
+/.git-blame-ignore-revs @stv0g
+/.github @m-mirz @leonardocarreras @stv0g
+/.gitignore @stv0g @m-mirz
+/cmake @m-mirz @stv0g
+/cmake/FindMAGMA.cmake @n-eiling
+/configs @m-mirz
+/configs/shmem_cosim/ @stv0g
+/docs @m-mirz
+/dpsim @m-mirz
+/dpsim-models @m-mirz
+/dpsim-villas/ @stv0g
+/dpsim/include/dpsim/GpuDenseAdapter.h @n-eiling
+/dpsim/include/dpsim/GpuMagmaAdapter.h @n-eiling
+/dpsim/include/dpsim/GpuSparseAdapter.h @n-eiling
+/dpsim/include/dpsim/MNASolverDynInterface.h @n-eiling
+/dpsim/include/dpsim/MNASolverFactory.h @n-eiling
+/dpsim/include/dpsim/MNASolverPlugin.h @n-eiling
+/dpsim/include/Utils.h @n-eiling
+/dpsim/src/GpuDenseAdapter.cpp @n-eiling
+/dpsim/src/GpuMagmaAdapter.cpp @n-eiling
+/dpsim/src/GpuSparseAdapter.cpp @n-eiling
+/dpsim/src/MNASolverPlugin.cpp @n-eiling
+/dpsim/src/pybind @stv0g
+/dpsim/src/SolverPlugins @n-eiling
+/dpsim/src/Utils.h @n-eiling
+/examples @m-mirz
+/examples/villas @stv0g
+/examples/villas-deprecated @stv0g
+/gitlab-ci.yml @stv0g
+/packaging @m-mirz
+/packaging/Docker @stv0g
+/packaging/Docker/Dockerfile.dev-rocky @n-eiling
+/pyproject.toml @stv0g
+/pytest.ini @stv0g  ?
```

Comment:
there is an extra `?` here, maybe is still pending someone else

Reply:
Fixed

Reply:
@stv0g I still see the old version…

Reply:
Oh sorry, I missed to push my changes